scripts/rook.sh

```diff
@@ -116,6 +117,22 @@ function delete_block_pool() {
 	rm -f "./newpool.yaml"
 }
 
+function create_block_ec_pool() {
+	curl -o block-pool-ec.yaml "${ROOK_URL}/pool-ec.yaml"
+	sed -i "s/ec-pool/$ROOK_BLOCK_EC_POOL_NAME/g" block-pool-ec.yaml
```

Comment:
variables are used with `${..}` in this script, use `${ROOK_BLOCK_EC_POOL_NAME}` here and at other places

Reply:
Thanks! Updated now